Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby File.read(File.join(File.dirname(__FILE__), '.ruby-version')).strip
+ruby File.read(File.join(File.dirname(__FILE__), '.ruby-version'))[/(\d\.){2}\d/]
```

Comment:
What's wrong with original `.strip`, what this regexp does better? It does that same, but the code is complex.

Reply:
> It does that same
Absolutely not.
I tested the application on intermediate versions of ruby, the old code does not allow me to freely switch between them, unlike this implementation
```
  2.5.0-dev
  2.5.0-preview1
  2.5.0-rc1
```

Reply:
Okay, I see the motivation behind it. My opinion is that it's good for this boilerplate to only allow stable releases to be used. If you want to use unstable, experimental, and other releases then you can change this in your fork/clone/repo, and this change will be explicit. So please rollback this version selection mechanism to the original one. Ruby 2.5.0 already released, should be no problems. Overall this PR is good!

Reply:
@zharikovpro This code allows to improve the flexibility of the application, considering that this is a template, this is an important and necessary, in my opinion, improvement, please reconsider your decision

Reply:
Your decision allows more flexibility with the price of security and stability. My Gemfile acts as a safety net when it doesn't allow you to install unstable version of ruby easily. This boilerplate is intended to be production-ready, and making another decision will break this.

I'm not gonna trade flexibility for security and stability. You can do this at your own risk outside of this repository.

Reply:
About flexibility: this boilerplate is already flexible just enough, as you can install any version easily, changing only two simple lines of code.

Reply:
@zharikovpro  I did not think that this was done intentionally, many thanks, now I understand